### PR TITLE
[SQL] Adding directories to separate new patches from clean-up patches from Archives

### PR DESCRIPTION
**Pull request by ridz1208**
This creates 2 new directories in the `/SQL/` folder: 
 - `Cleanup_patches`: Directory where release clean-ups should be made to. to replace putting SQL code in PHP tools scripts and/or commenting out clean-up commands in the release scripts (See README.md for more details)
 - `New_patches`: Directory where new SQL scripts go instead of putting them in Archives which should contain **archives** and not patches that have not yet been ran... This directory contains any and all new sql scripts no matter the branch (See README.md for more details)

### CODE REVIEW

**Comment by ridz1208** [comment]
@driusan please review grammar :S

File: SQL/Cleanup_patches/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This directory contains SQL patches written as a post-release clean-up. These patches can contain `ALTER TABLE`, `DELETE` and/or `DROP` statements.
+
```
Comment (PapillonMcGill) [inline SQL/Cleanup_patches/README.md]: Does that mean someone could include the following instruction in such cleanup patches?
ALTER TABLE ADD COLUMN ....

File: SQL/Cleanup_patches/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+These patches should be run at the discretion of the projects with the full knowledge of the possible side effects of these.
```
Comment (PapillonMcGill) [inline SQL/Cleanup_patches/README.md]: What would happen if a project decide not to run those cleanup patches?

File: SQL/Cleanup_patches/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+This directory contains SQL patches written as a post-release clean-up. These patches can contain `ALTER TABLE`, `DELETE` and/or `DROP` statements.
```
Comment (driusan) [inline SQL/Cleanup_patches/README.md]: Can we call it CleanupPatches instead of `Cleanup_patches`? I don't like the mixed case and underscore.

Comment (driusan) [reply SQL/Cleanup_patches/README.md]: s/as a/for/

Comment (driusan) [reply SQL/Cleanup_patches/README.md]: s/clean-up/cleanup/

File: SQL/Cleanup_patches/README.md (hunk 2 above)
Comment (driusan) [inline SQL/Cleanup_patches/README.md]: "of the patches" or "of these patches"

Comment (driusan) [reply SQL/Cleanup_patches/README.md]: I think this is also part of the first paragraph and not a second paragraph too

File: SQL/New_patches/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+This directory contains SQL patches necessary during the upgrade of loris. Upon release, the SQL patches concerned will be concatenated into a single release patch stored under the `SQL/Release_patches` directory. The files used for the concatenation are then moved to the `SQL/Archive/X.Y` directory.
```
Comment (driusan) [inline SQL/New_patches/README.md]: same comment about the directory name, and LORIS should be uppercase.

File: SQL/Cleanup_patches/README.md (hunk 2 above)
Comment (PapillonMcGill) [reply SQL/Cleanup_patches/README.md]: I like "of these patches"

Comment (driusan) [reply SQL/Cleanup_patches/README.md]: Another option is to just leave off the "of these"

File: SQL/Cleanup_patches/README.md (hunk 1 above)
Comment (ridz1208) [reply SQL/Cleanup_patches/README.md]: @PapillonMcGill they could include whatever they want as long as its not critical to the functioning of the code.

things that go here are things that usually are commented out in the release patches

File: SQL/Cleanup_patches/README.md (hunk 2 above)
Comment (ridz1208) [reply SQL/Cleanup_patches/README.md]: theoretically everything should still work fine in their codebase/database, these should only be clean-up

File: SQL/Cleanup_patches/README.md (hunk 3 above)
Comment (ridz1208) [reply SQL/Cleanup_patches/README.md]: i was just being consistent with Release_patches.

I dont mind the change if you approve having `Release_patces` and `CleanupPatches` in the same directory

File: SQL/Cleanup_patches/README.md (hunk 2 above)
Comment (ridz1208) [reply SQL/Cleanup_patches/README.md]: added a line to explain

File: SQL/Cleanup_patches/README.md (hunk 1 above)
Comment (ridz1208) [reply SQL/Cleanup_patches/README.md]: added a line to explain

**Comment by ridz1208** [comment]
@driusan @PapillonMcGill please re-review

File: SQL/Cleanup_patches/README.md [hunk 5]
```diff
@@ -1,3 +1 @@
-This directory contains SQL patches written for post-release cleanup. These patches can contain `ALTER TABLE`, `DELETE` and/or `DROP` statements.
-
-These patches should be run at the discretion of the projects with the full knowledge of the possible side effects of these patches.
+This directory contains SQL patches written for post-release cleanup. These patches can contain `ALTER TABLE`, `DELETE` and/or `DROP` statements. These patches should be run at the discretion of the projects with the full knowledge of the possible side effects of these patches. The content of these patches should not be mandotory for proper functioning of the database, they should be run for the sole purpose of removing unused tables, columns or fields; any mandatory cleanup should be added to the release patches.
```
Comment (driusan) [inline SQL/Cleanup_patches/README.md]: 👍 +1

File: SQL/Cleanup_patches/README.md [hunk 6]
```diff
@@ -0,0 +1 @@
...
```
Comment (PapillonMcGill) [inline SQL/Cleanup_patches/README.md]: You mention DELETE statement but near the end of the paragraph, you mention removing columns or fields, not rows.

Could you add precision about when DELETE could be used? or remove DELETE statement?

Comment (ridz1208) [reply SQL/Cleanup_patches/README.md]: rows should be there as well, adding it to the  readme

**Pull request merged**